packages/grid/x-data-grid-pro/src/DataGridPro/useDataGridProProps.ts

```diff
@@ -25,6 +26,7 @@ export const DATA_GRID_PRO_PROPS_DEFAULT_VALUES: DataGridProPropsWithDefaultValu
   disableChildrenFiltering: false,
   disableChildrenSorting: false,
   rowReordering: false,
+  rowsLoadingMode: GridFeatureModeConstant.client,
```

Comment:
I would personally use a typed string (`type GridFeatureModeConstant = 'client' | 'thing1' | 'thing2'` instead of an enum to be more flexible and avoid forcing an import.